frontend/packages/ux-editor/src/components/Properties/Text.tsx

```diff
@@ -75,15 +76,12 @@ export const Text = () => {
           <Heading level={2} size='2xs' className={classes.heading}>
             {t('ux_editor.properties_panel.texts.sub_title_images')}
           </Heading>
-          <EditImage
-            component={form}
-            handleComponentChange={async (updatedComponent) => {
-              handleUpdate(updatedComponent);
-              debounceSave(formId, updatedComponent);
-            }}
-          />
+          <EditImage component={form} handleComponentChange={handleComponentChange} />
         </>
       )}
+      {form.type === 'SubForm' && (
```

Comment:
Can we use `ComponentType.SubForm` here?

Reply:
Nice catch! 😄 
Its updated now